packages/plugin-redis-core/src/migrate.ts

```diff
@@ -81,6 +81,33 @@ const appendLuaScript = (finalVersion: number, min = 0, script: string) => [
   appendPostScript(finalVersion),
 ].join('\n')
 
+const getMigrationFile = async (script: Path) => {
+  let file = script.fullpath()
+
+  if (script.isDirectory()) {
+    const files = await script.readdir()
+    const availableExtensions = []
+
+    for (const item of files) {
+      const parsed = path.parse(item.fullpath())
+      if (parsed.name === 'index') {
+        availableExtensions.push(parsed.ext)
+      }
+    }
+
+    for (const extension of ['js', 'ts', 'cjs', 'mjs']) {
```

Comment:
It might make sense to search for files in reverse order ['ts', 'js']

Reply:
alternatively `const isTypeScript = __filename.endsWith('ts')`

Reply:
__filename is not available in MJS